db/sqlite_store.py

```
@@ -580,6 +709,14 @@ async def _init_schema(self) -> None:
         log.info("SQLiteStore schema ready at %s", self._db_path)
 
     async def close(self) -> None:
+        if self._read_pool is not None:
+            while not self._read_pool.empty():
+                rc = self._read_pool.get_nowait()
+                try:
+                    await rc.close()
+                except Exception:  # pragma: no cover - best effort
+                    pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)